website/docs/docs/build/saved-queries.md

```diff
@@ -17,8 +17,32 @@ To create a saved query, refer to the following table parameters.
 :::tip
 Note that we use the double colon (::) to indicate whether a parameter is nested within another parameter. So for example, `query_params::metrics` means the `metrics` parameter is nested under `query_params`.
 :::
+
+<!-- For versions 1.9 and higher -->
+<VersionBlock firstVersion="1.9">
```

Comment:
I didn't review this as I'm not sure what has changed and it doesn't seem related to tags

Reply:
good shout! just added that `tags` are supported in 1.9+ under the `config` value:
![Screenshot 2024-12-23 at 11 43 39](https://github.com/user-attachments/assets/840658b9-f8a4-4349-b9ee-c1389f7914c6)